src/modules/xml/producer_xml-clip.yml

```diff
@@ -0,0 +1,18 @@
+schema_version: 0.1
```

Comment:
I prefer to use the latest version 7.0 for new files,  partly because I feel that since this is a new producer (not a specially named producer for the existing `xml` producer) and should get the 
```
parameters:
  - identifier: resource
    argument: yes
    title: File
    type: string
    description: An XML text file containing MLT XML.
    readonly: no
    required: yes
    mutable: no
    widget: fileopen
```
And `argument` requires a newer schema version than the first 0.1. Instead of analyzing the minimum version just use the latest to not worry.

Reply:
Done